docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,180 @@
+
+[[bxms_rn_whats_new]]
+= What's New
+
+This section highlights new features in {PRODUCT} {PRODUCT_VERSION} Limited Availability.
+
+== Business Central
+
+=== Improved Navigation
+
+.Navigation between views
+You can now navigate as follows:
+
+* From a process instance to the associated jobs
+* From a job to the associated process instance
+* From a process instance to the associated tasks
+
+.Business Central home page
+The Business Central Home page has been redesigned for easier navigation. For more information, see the "Business Central" chapter of the _{PRODUCT} User Guide_.
+
+=== Guided Decision Table
+* Guided Decision Table has a new wizard for creating and editing columns.
+* Hit policies are now available. Hit policies determine the order in which rules (rows) in a guided decision table are applied.
+* New support to display multiple linked guided decision tables in the Guided Decision Table Editor has been added.
+
+=== Project Metrics Dashboard
+A new dashboard is now available for every project listed in the authoring library. 
```

Comment:
@emmurphy1 The old description of this feature was much longer. Was it reduced on purpose?

Reply:
@sutaakar Yes, Christy asked me to summarize all of the entries.